### PR TITLE
Update the repo README to be the team faq and directory

### PR DESCRIPTION
**Pull request by bravefoot**
Hopefully this is the right place. If we want to move this info to a subdirectory, let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
-# DevOps
+# Helpful Engineering DevOps Team
 
-Repository for automation scripts.
+## FAQ
+
+**How do I get help?**
```
Comment (yardensachs) [inline README.md]:
```suggestion
### How do I get help?
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+
+Click the lightning bolt in the header of the slack channel and select “Help Request”
+
+**How do I help out?**
```
Comment (yardensachs) [inline README.md]:
```suggestion
### How do I help out?
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+
+Fill out our skill survey at https://airtable.com/shrisE8yL9W3cegHr and check out our Trello board at https://trello.com/invite/b/8TEQ0gH0/82b3cce6da639b5cebde085def1d2ede/skill-software-devops-projects
+
+**When are check-in meetings?**
```
Comment (yardensachs) [inline README.md]:
```suggestion
### When are check-in meetings?
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+
+Daily at 1500 PT/2200 GMT on zoom: https://zoom.us/j/471016936?pwd=cXh2RWFib2F4RU1ldVZPM3BNdkcrQT09
+
+**Who should I ask about X?**
```
Comment (yardensachs) [inline README.md]:
```suggestion
### Who should I ask about X?
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,35 @@
...
+
+Check out the directory below
+
+##Directory
```
Comment (yardensachs) [inline README.md]:
```suggestion
## Directory
```

**Review by yardensachs — COMMENTED** [review]
to be considered https://github.com/helpfulengineering/devops/issues/1

File: README.md [hunk 6]
```diff
@@ -1,3 +1,35 @@
...
+
+### Services
+
+- AWS: @Chris Mac @skuenzli
+- Azure: @Jonathan Vermeij @Troy Witthoeft
+- Central project & volunteer list/AirTable: @psprings @jonnyparris
```
Comment (chris-mac) [inline README.md]: Think @ken should be added here. Worth checking  with @psprings that @jonnyparris is working on the same system

**Comment by chris-mac** [comment]
@bravefoot if we get those formatting suggestions from @yardensachs added and the update to add @ken I think we are good to go. Better to just get first draft in and we can update as we go.

**Comment by dmangot** [comment]
:shipit:

**Pull request merged**